### PR TITLE
Add a select helper to MemberGroup

### PR DESCRIPTION
**Pull request by YunzheZJU**
#### TODO
- [x] I18n for ja.

#### Preview

* If you mute 2 members in a sequence, you will be asked whether to toggle off all the members in the group. You are free to click 'No' or just ignore it.

* If you confirm the operation, all members will be muted. You are given a change to roll back the changes.

* If you subscribe 2 members in a sequence, you will be asked whether to toggle on all the members in the group.

### CODE REVIEW

File: src/options/i18n/locales/ja.json5 [hunk 1]
```diff
@@ -28,5 +28,20 @@
...
+      // Prompt users to subscribe to all members in a group or not, with the question mark
+      subscribe: "TODO:Subscribe to this group?",
+      // Indicate that the requested group operation has succeeded, with the period
+      success: "TODO:Operations performed.",
+      // Confirm the operation
+      yes: "TODO:Yes",
+      // Cancel the operation
+      no: "TODO:No",
+      // Undo the operation
+      undo: "TODO:Undo"
```
Comment (YunzheZJU) [inline src/options/i18n/locales/ja.json5:44]: This pull-request needs translations in Japanese for these strings marked with `TODO:`.
Please make a comment, propose a suggestion here or make another pull-request to this branch `select-helper` if you are interested.

File: src/options/i18n/locales/ja.json5 [hunk 2]
```diff
@@ -28,5 +28,20 @@
   memberGroup: {
     // Indicate that no member data can be found for this group
     noData: "メンバーのデータがありません",
+    // Help users to perform group operations when detected
+    selectHelper: {
+      // Prompt users to unsubscribe to all members in a group or not, with the question mark
+      mute: "TODO:Mute this group?",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:34]:
```suggestion
      mute: "このグループをミュートしますか？",
```

File: src/options/i18n/locales/ja.json5 [hunk 3]
```diff
@@ -28,5 +28,20 @@
...
+      // Prompt users to subscribe to all members in a group or not, with the question mark
+      subscribe: "TODO:Subscribe to this group?",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:36]:
```suggestion
      subscribe: "このグループを登録しますか？",

File: src/options/i18n/locales/ja.json5 [hunk 4]
```diff
@@ -28,5 +28,20 @@
...
+      // Indicate that the requested group operation has succeeded, with the period
+      success: "TODO:Operations performed.",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:38]:
```suggestion
      success: "設定しました。",
```

File: src/options/i18n/locales/ja.json5 [hunk 5]
```diff
@@ -28,5 +28,20 @@
...
+      // Confirm the operation
+      yes: "TODO:Yes",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:40]:
```suggestion
      yes: "はい",
```

File: src/options/i18n/locales/ja.json5 [hunk 6]
```diff
@@ -28,5 +28,20 @@
...
+      // Cancel the operation
+      no: "TODO:No",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:42]:
```suggestion
      no: "いいえ",
```

File: src/options/i18n/locales/ja.json5 (hunk 1 above)
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:44]:
```suggestion
      undo: "戻る"
```

**Review by ReiRokusanami0010 — COMMENTED** [review]
I translated it again this time. Please review it.  ;D

**Comment by YunzheZJU** [comment]
> I translated it again this time. Please review it.  ;D

ありがとうございます😊！助かりました！！！！！！！！！！！

**Pull request merged**